### PR TITLE
Assessment tool output csv feature

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added csv output for host info and benchmarks results

### CODE REVIEW

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: cmd/assessment/benchmarks/arwenBenchmark.go [hunk 1]
```diff
@@ -42,6 +43,11 @@ func NewArwenBenchmark(arg ArgArwenBenchmark) *arwenBenchmark {
 
 // Run returns the time needed for the benchmark to be run
 func (ab *arwenBenchmark) Run() (time.Duration, error) {
+	err := fileExists(ab.scFilename)
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply cmd/assessment/benchmarks/arwenBenchmark.go:46]: didn't knew about that function. Thanks!

[the remark this answers is not in the document]
Comment (iulianpascalau) [reply cmd/assessment/benchmarks/arwenBenchmark.go:46]: done

File: cmd/assessment/benchmarks/delegationBenchmark.go [hunk 2]
```diff
@@ -40,6 +40,11 @@ func NewDelegationBenchmark(arg ArgDelegationBenchmark) *delegationBenchmark {
 
 // Run returns the time needed for the benchmark to be run
 func (db *delegationBenchmark) Run() (time.Duration, error) {
+	err := fileExists(db.scFilename)
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply cmd/assessment/benchmarks/delegationBenchmark.go:43]: done

File: cmd/assessment/benchmarks/erc20Benchmark.go [hunk 3]
```diff
@@ -37,6 +37,11 @@ func NewErc20Benchmark(arg ArgErc20Benchmark) *erc20Benchmark {
 
 // Run returns the time needed for the benchmark to be run
 func (eb *erc20Benchmark) Run() (time.Duration, error) {
+	err := fileExists(eb.scFilename)
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply cmd/assessment/benchmarks/erc20Benchmark.go:40]: done

File: cmd/assessment/benchmarks/results.go [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+	Results       []SingleResult
+}
+
+// ToDisplayTable will output the contained data as an ASCII table
+func (tr *TestResults) ToDisplayTable() string {
+	hdr := []string{"Benchmark", "Time in seconds", "Error"}
+	lines := make([]*display.LineData, 0, len(tr.Results)+1)
+	for i, res := range tr.Results {
+		errString := ""
+		if res.Error != nil {
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply cmd/assessment/benchmarks/results.go:32]: done

File: cmd/assessment/benchmarks/errors.go [hunk 5]
```diff
@@ -7,3 +7,6 @@ var ErrEmptyBenchmarksSlice = errors.New("empty benchmarks slice provided")
 
 // ErrNilBenchmark signals that a nil benchmark was provided
 var ErrNilBenchmark = errors.New("nil benchmark")
+
+// ErrFileDoesNotExists signals that the required file does not exists
+var ErrFileDoesNotExists = errors.New("file does not exists")
```
Comment (bogdan-rosianu) [inline cmd/assessment/benchmarks/errors.go:12]: does not exist*

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Pull request merged**